src/components/activities/RosterPanel.tsx

```diff
@@ -136,14 +137,15 @@ function ParticipantHoursText({ participant }: { participant: Participant }) {
   }, [lastTimeline, latestTimeout, participant.firstname]);
 
   let timeOnClock = 0;
-  let timeout = latestTimeout;
+  let lastTime: number = latestTimeout;
   for (const t of participant.timeline) {
     if (isOnClock(t.status)) {
-      timeOnClock += timeout - t.time;
-    } else {
```

Comment:
Basically other than the rename I think you just got rid of the "else"?

Reply:
Ya, I guess so! Conceptually went from always using the last time out in the subtraction to just using the last time. Practically that just meant removing the else.